jsk_arc2017_common/config/box_sizes.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
So, below is the final directory construction?
It looks that `config` is for small data, `data` is for large data.
How do you chose which directory you use? @knorth55 @pazeshun 

- config/classes.txt
- config/grasp_types.txt
- config/label_names.yaml
- config/box_sizes.json
- config/shef_bin.yaml
- config/shelf_interactive_marker.yaml
- config/objects/
- data/datasets/
- data/models/
- data/others/

I don't understand the why about below comments: @knorth55 @pazeshun 

> data is too big dir. Important info should be found in config
> image files and datasets are not important, but probably yes.

datasets and models are not important?

> env setting includes: ...

datasets and models are not information of env?
Do you mean files for `rosparam load` is information of env?